jwt-bcp/draft-sheffer-oauth-jwt-bcp.md

```diff
@@ -191,19 +325,42 @@ may sign JWTs with different "typ" values:
 * "JWT+set+oidc-logout" for OID Connect Logout events.
 
 Specifications that define JWT applications SHOULD define a unique "typ" value.
+-->
+
+Each application of JWTs defines a profile specifying the required and optional JWT claims
+and the validation rules associated with them.
+To prevent substitution of JWTs from one context into another, a number of strategies may be employed:
```

Comment:
Again, we disagree here. IMO multiple different strategies will not work. We need a single strategy for everyone to agree on, otherwise we leave ourselves open to token confusion attacks.